README.md

```diff
@@ -2,9 +2,39 @@
 
 > This package includes the Babel preset used by Enact.
 
+## Usage
+
+Reference this module from your `babel.config.js`.
+
+```js
+module.exports = require('babel-preset-enact');
```

Comment:
I think CRA describes well with this. Would you please add `Usage in Enact projects created by Enact CLI` and `Usage Outside of Enact projects`?
Also, it would be better to present usage with `preset` instead of this line.

Reply:
OK